bikeshed/__init__.py

```diff
@@ -1248,6 +1262,7 @@ def preprocess(self):
         processHeadings(self, "all") # again
         addTOCSection(self)
         addSelfLinks(self)
+        processBiblioLinks(self) # again, as biblio links could be added by addIndexSection().
```

Comment:
We need to run it before we run `addReferencesSection`. I'll see if I can find an ordering that works. If you don't mind reversing the references section and the index, we could do `addIndex -> processBiblio -> addReferences`?